src/stores/Store.js

```diff
@@ -51,4 +51,26 @@ export class Store {
     this.courseModalState = new CourseModalState(this.mainPlan);
   }
 
+  @action.bound loadPlan(id) {
+    fetch('/api/plan/load?planId='+id)
+      .then((response) => response.json())
+      .then((data) => {
+        this.mainPlan = deserialize(PlanModel, data, () => {});
+      });
+  }
+
+  @action.bound savePlan() {
+    fetch('/api/plan/save', {
+      method: 'post',
+      headers: {
```

Comment:
i feel like its obnoxious to have to put in the headers on every post request like this

Reply:
we can pull then out into a constant?

Reply:
hmmm might be a good idea... lets do that another time though